<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PasswordDialog</class>
 <widget class="QDialog" name="PasswordDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>340</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>450</width>
    <height>280</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>450</width>
    <height>350</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Poppins</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Please Enter Your Password</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="minimumSize">
      <size>
       <width>450</width>
       <height>340</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>450</width>
       <height>340</height>
      </size>
     </property>
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #282d31; 
border-radius: 5px;
</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLineEdit" name="m_passwordEdit">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>160</y>
        <width>271</width>
        <height>40</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">padding: 3px; border: 1px solid #555; border-radius: 5px;   color: #aaa;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="echoMode">
       <enum>QLineEdit::Password</enum>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="placeholderText">
       <string>Type your password...</string>
      </property>
     </widget>
     <widget class="QPushButton" name="b1_okButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>230</y>
        <width>271</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>125</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">/* This style is handled automatically by EditableStyle.cpp */</string>
      </property>
      <property name="text">
       <string>CONTINUE</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="m_errorLabel">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>371</width>
        <height>31</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>122</width>
        <height>0</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: orange; font-size: 16px; border: none;</string>
      </property>
      <property name="text">
       <string>Incorrect Password</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>381</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
        <pointsize>-1</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #fff;
border-right: 0px;
font-size: 20px;</string>
      </property>
      <property name="text">
       <string>Please Enter Your Password</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="m_currentWalletTitle">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>76</y>
        <width>371</width>
        <height>26</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Roboto</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: #aaa;
border-right: 0px;</string>
      </property>
      <property name="text">
       <string notr="true">currentWallet</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="b1_cancelButton">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>280</y>
        <width>271</width>
        <height>40</height>
       </rect>
      </property>
      <property name="minimumSize">
       <size>
        <width>125</width>
        <height>25</height>
       </size>
      </property>
      <property name="font">
       <font>
        <family>Poppins</family>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">/* This style is handled automatically by EditableStyle.cpp */</string>
      </property>
      <property name="text">
       <string>CANCEL</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>b1_okButton</sender>
   <signal>clicked()</signal>
   <receiver>PasswordDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>158</x>
     <y>68</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>b1_cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>PasswordDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>225</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>224</x>
     <y>169</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
